quickstarts/deeper-network/dashboards/deeper-connect.json

```diff
@@ -1,7 +1,6 @@
 {
     "name": "Deeper Network - Connect Monitoring",
     "description": null,
-    "permissions": "PUBLIC_READ_WRITE",
```

Comment:
@jpvajda @aswanson-nr this is another manual step that Kav had to do after exporting a dashboard from within NR1. How could we make this easier for contributors so they don't have to do this manually?

Reply:
If it helps, whilst I was PR'ing the related [entity synthesis](https://github.com/newrelic/entity-definitions#dashboard-templates) for this, their repo contains a `sanitize dashboards` function that runs via docker compose. This handled everything for me, would be nice to have similar.

Reply:
@Kav91 Great suggestion! I'll dig into that and get a ticket created for us.

Reply:
yes, that was what @thezackm also mentioned we could use

Reply:
@aswanson-nr I found this old issue which seem to be about this problem

https://github.com/newrelic/newrelic-quickstarts/issues/55